mdb/mdb_launch.py

```diff
@@ -56,10 +66,23 @@
     help="Starting port address. Each rank's port is assigned as [port_address + rank].",
 )
 @click.option(
-    "--config-filename",
-    default=".mdb.conf",
+    "-b",
+    "--backend",
+    default="gdb",
+    help="Debug backend e.g., gdb, lldb etc.",
+)
+@click.option(
+    "-t",
+    "--target",
+    type=click.File("r"),
+    required=True,
+    help="Target binary to debug.",
+)
+@click.option(
+    "--mdb-home",
+    default="~/.mdb",
     show_default=True,
-    help="filename for the mpirun configuration.",
+    help="filename for the mpirun appfile.",
```

Comment:
Is this correct for the `--mdb-home` options? I thought that was where e.g. the certs and keys are stored?

Reply:
:flushed: nope. I have fixed this now